### PR TITLE
Fix scripts error handling

### PR DESCRIPTION
**Pull request by facuspagnuolo**
As we've been saying before, we do need to refactor these deployment scripts, there's code duplicated and it is not working flawless. This PR does not solve that at all, just tweaking it to at least not swallow thrown errors. 

I'm not aware of how the CLI interacts with these scripts, or if it expects them to behave in certain way. Note that I'm changing the way the truffle exec callback is handled in the scripts.

### CODE REVIEW

File: kits/democracy/scripts/deploy.js [hunk 1]
```diff
@@ -22,5 +27,9 @@ module.exports = async (callback) => {
   // aragonID
   await deploy_id(null, { artifacts, web3, ensAddress: ens.address })
 
-  await deploy_kit(null, { artifacts, kitName: 'democracy-kit', kitContractName: 'DemocracyKit', network: network, ensAddress: ens.address })
+  await deploy_kit(null, { artifacts, kitName: 'democracy-akropolis', kitContractName: 'DemocracyKit', network: network, ensAddress: ens.address })
```
Comment (bingen) [inline kits/democracy/scripts/deploy.js]: Just curious, why this change? Anyway, it's definitely nicer! ;-) (although maybe less intuitive, and I see mltisig one hasn't been changed)

Comment (facuspagnuolo) [reply kits/democracy/scripts/deploy.js]: My bad! 🤦🏼‍♂️
I carried a change from another fork, please merge #81 to fix it 🙏🏻
